pkg/restapi/operation/operations.go

```diff
@@ -35,6 +36,10 @@ const (
 
 var logger = log.New("hub-auth-restapi")
 
+type oidcClaims struct {
+	Sub string `json:"sub"`
```

Comment:
and here

Reply:
This is directly related to the id_token claim I'm trying to extract, so doesn't make sense to change

Reply:
OK 👍